entries/g/guildwars2.com.json

```
@@ -1,14 +1,19 @@
 {
   "Guild Wars 2": {
     "domain": "guildwars2.com",
+    "additional-domains": [
+      "arena.net"
+    ],
     "url": "https://www.guildwars2.com",
     "tfa": [
+      "email",
       "sms",
       "totp"
     ],
     "documentation": "https://help.guildwars2.com/hc/en-us/articles/230672927",
+    "recovery": "https://help.guildwars2.com/hc/en-us/requests/new?ticket_form_id=360003248533",
```

```suggestion
    "recovery": "https://help.guildwars2.com/hc/en-us/articles/360000819168",
```
Usually we prefer to link to the 2FA recovery documentation, rather than the support form